demos/gesture_recognition_demo/cpp_gapi/create_list.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+'''
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+'''
+
+import glob
+import os
+import json
+import argparse
+
+parser = argparse.ArgumentParser(description='This is')
```

Comment:
```suggestion
parser = argparse.ArgumentParser()
```
That `This is` looks weird by itself.

Reply:
Removed.

Reply:
You shouldn't keep `description=''`. if we don't change it, then one, who reads the demo code, shouldn't bother, well, reading this.

Reply:
Removed.